Frends.Salesforce.DeleteSObject/Frends.Salesforce.DeleteSObject/Frends.Salesforce.DeleteSObject.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-    <Version>2.0.0</Version>
+    <Version>2.0.1</Version>
```

Comment:
Version increases should go to second version number, so instead of 2.0.1 it should be 2.1.0